src/scss/includes/components/button.scss

```diff
@@ -37,13 +42,16 @@
   }
 
   &-icon + &-content {
-    margin-left: 6px;
+    margin-left: 4px;
   }
 
   &::-moz-focus-inner {
     border: 0;
   }
 
+  &:not(.button--primary):hover {
+    background: $action-lighter;
```

Comment:
On touch devices we can add `:active` to avoid the hover state getting stucked once clicked